packages/health-and-info/src/index.ts

```diff
@@ -0,0 +1,195 @@
+import AgentOptions, { HttpOptions, HttpsAgent, HttpsOptions } from 'agentkeepalive'
```

Comment:
**suggestion:** `AgentOptions` isn't really a set of options, maybe `Agent` or `HttpAgent` would be better?

Reply:
Good suggestion, superagent's `.agent` chained function takes SARAgent, http.Agent or https.Agent, so instead it would be ideal to drop AgentOptions and switch to
```
    const agent = component.url.startsWith('https')
      ? new Https.Agent(component.agentConfig as HttpsOptions)
      : new Http.Agent(component.agentConfig as HttpOptions)
 ```

Reply:
👍  
needs to be the `agentkeepalive` version of the agent rather than the standard http one

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-typescript-lib/pull/7/commits/db83b30a3f287b275ffe3e5f7f5fdaade66684ae